nautobot/core/tests/__init__.py

```diff
@@ -0,0 +1,9026 @@
+PERFORMANCE_BASELINES = {
```

Comment:
Would it make more sense to store this as a separate YAML file rather than inline as code?